### PR TITLE
Added Input File Options to the hook and openFileDialog function

### PR DESCRIPTION
**Pull request by m-brtn**
interface Options {
	multiple?: boolean;
	accept?: string;
}

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,40 +1,48 @@
...
-  },
-  "peerDependencies": {
-    "react": ">=16.3",
-    "react-dom": ">=16.3"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/omidnikrah/use-file-dialog.git"
-  }
+	"name": "use-file-dialog",
```
Comment (omidnikrah) [inline package.json]: Could you please revert your changes to this file? because only the spaces were changed.

**Comment by omidnikrah** [comment]
Thank you so much @ruslanchek. That's a great PR.

File: package.json (hunk 1 above)
Comment (m-brtn) [reply package.json]: I reverted

**Comment by willurmston** [comment]
@omidnikrah Hey, could you publish these changes please? Love the simplicity of this hook!

**Comment by omidnikrah** [comment]
Hey @willurmston, of course, I'll do it ASAP.

**Pull request merged**